.github/workflows/build.yml

```diff
@@ -187,3 +187,19 @@ jobs:
         run: make init
       - name: Run functional & smoke tests
         run: pytest -vv -n 4 tests/functional tests/smoke
+
+  docker-disabled:
+    name: docker-disabled-test
+    if: github.repository_owner == 'aws'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+        with:
+          python-version: "3.7" # The integration test below needs python3.7 be to configured on the PATH
```

Comment:
aside: need to update the python version on the templates in general.

Reply:
I'll update to a newer version, just want to figure out the GHA stuff first 🙃